addon/components/modal-component.js

```diff
@@ -250,6 +252,8 @@ ModalComponent.reopenClass({
   hideAll() {
     return $(document).trigger('modal:hide');
   },
+
+  // TODO Deprecate this old API
```

Comment:
As I believe Addepar are the only ones using this library (there are no npm releases for example) I think we can be liberal changing the API without a deprecation phase.

Reply:
Sounds good. I'm replacing it with a `// TODO remove old API`